miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -20,6 +20,11 @@ abstract class MiniAppMessageBridge {
         callback: (isGranted: Boolean) -> Unit
     )
 
+    /** This host app info can apply to analytics, app traffic, ads, etc.
```

Comment:
I think the documentation needs to mention that this is being appended to the User Agent.